valhalla/baldr/edgeinfo.h

```diff
@@ -189,11 +197,12 @@ class EdgeInfo {
   // 1st 8-byte word
   union Word0 {
     struct {
-      uint64_t wayid_ : 32;          // OSM way Id
-      uint64_t mean_elevation_ : 12; // Mean elevation with 2 meter precision
-      uint64_t bike_network_ : 4;    // Mask of bicycle network types (see graphconstants.h)
-      uint64_t speed_limit_ : 8;     // Speed limit (kph)
-      uint64_t spare0_ : 8;
+      uint64_t wayid_ : 32;                // OSM way Id
+      uint64_t mean_elevation_ : 12;       // Mean elevation with 2 meter precision
+      uint64_t bike_network_ : 4;          // Mask of bicycle network types (see graphconstants.h)
+      uint64_t speed_limit_ : 8;           // Speed limit (kph)
+      uint64_t speed_limit_unlimited_ : 1; // Unlimited speed limit
```

Comment:
instead of taking another bit, could we just use 257 to signify unlimited. i would rather steal one (probably unused value) from the existing speed limit than steal another bit from spare

Reply:
That's the approach we discussed first, but it seemed like that would have the side effect of setting the edge speed to the same high value. If we can do it without side effects though I would prefer that approach.

Reply:
@gknisely @kevinkreiser I think I got this working, but are we okay with returning `speed_limit: 255` in the serialized response in this case?
![image](https://user-images.githubusercontent.com/5665333/75178566-5672b600-5706-11ea-97e9-05be6cc7a104.png)